docs/canvas/canvas-expression-lifecycle.asciidoc

```diff
@@ -0,0 +1,259 @@
+[role="xpack"]
+[[canvas-expression-lifecycle]]
+== Canvas expression lifecycle
+
+Elements in Canvas are all created using an *expression language* that defines how to retrieve, manipulate, and ultimately visualize data. The goal is to allow you to do most of what you need without understanding the *expression language*, but learning how it works unlocks a lot of Canvas's power.
+
+
+[[canvas-expressions-always-start-with-a-function]]
+=== Expressions always start with a function
+
+Expressions simply execute <<canvas-function-reference, functions>> in a specific order, which produce some kind of output. That output can then be inserted into another function, and another after that, until it produces the output you need. 
```

Comment:
```suggestion
Expressions simply execute <<canvas-function-reference, functions>> in a specific order, which produce some output value. That output value can then be *piped* into another function, and another after that, until it produces the output you need. 
```

Reply:
I'll take your first suggestion, we made a design decision to use "inserted" instead of "piped" in the docs, as some users may not understand that verbiage.